presto-main/src/main/java/com/facebook/presto/execution/executor/TaskExecutor.java

```diff
@@ -168,8 +167,14 @@ public TaskExecutor(int runnerThreads, int minDrivers, Ticker ticker)
 
     @VisibleForTesting
     public TaskExecutor(int runnerThreads, int minDrivers, MultilevelSplitQueue splitQueue, boolean legacySchedulingBehavior, Ticker ticker)
+    {
+        this(runnerThreads, minDrivers, DEFAULT_MIN_SPLITS_PER_TASK, splitQueue, legacySchedulingBehavior, ticker);
```

Comment:
Given MIN_SPLITS_PER_TASK (and also max, which will soon be introduced in the next commit) is no longer a constant, I don't think it makes much sense to assign default values here. I would prefer that those two argument be mandatory in both `public TaskExecutor(int runnerThreads, int minDrivers, Ticker ticker)` and `public TaskExecutor(int runnerThreads, int minDrivers, MultilevelSplitQueue splitQueue, boolean legacySchedulingBehavior, Ticker ticker)`